sdk/core/core-http/src/policies/tracingPolicy.ts

```diff
@@ -86,11 +86,11 @@ export class TracingPolicy extends BaseRequestPolicy {
 
   tryCreateSpan(request: WebResourceLike): Span | undefined {
     try {
-      const path = URLBuilder.parse(request.url).getPath() || "/";
+      const scheme = URLBuilder.parse(request.url).getScheme() || "https";
```

Comment:
there is no need to distinguish HTTP and HTTPS  in span names, the otel spec recommends to use HTTP all the time. If somebody really cares about scheme, they can still look into url

Reply:
>therefore, HTTP client spans SHOULD be using conservative, low cardinality names formed from the available parameters of an HTTP request, such as "HTTP {METHOD_NAME}"


https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name

Reply:
Awesome, thanks! I changed it to always `HTTP {method}`. I put a comment explaining that this is as expected in core-rest-pipeline (didn't bother with core-http as its going away)